tests/VMelnalksnis.NordigenDotNet.Tests/Accounts/AccountClientTests.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+using NodaTime;
+
+using VMelnalksnis.NordigenDotNet.Accounts;
+using VMelnalksnis.NordigenDotNet.Tests.MockHttpResponse;
+
+namespace VMelnalksnis.NordigenDotNet.Tests.Accounts;
+
+public sealed class AccountClientTests
+{
+	public HttpClient MockHttpHandler(string json)
+	{
+		var client = new HttpClient(new HttpMessageHandlerMock(json));
+		client.BaseAddress = new("http://localhost");
+		return client;
+	}
+
+	public string GetTestData(string testPath)
+	{
+		var filePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, $"../../../Stubs/{testPath}");
+		return File.ReadAllText(filePath);
+	}
+
+	[Fact]
+	public async Task GetDetails_ShouldReturnExpected()
+	{
+		var data = GetTestData("Accounts/GetDetails.json");
+		var expected = new AccountDetails
+		{
+			CashAccountType = "OTHR",
+			Currency = "EUR",
+			Details = "A Sample Card Type",
+			Iban = "GL5117350000017350",
+			MaskedPan = "XXXX-XXXXXX-12345",
+			Name = "Credit Account",
+			OwnerName = "John Doe",
+			Product = "Card",
+			ResourceId = "01F3NS4YV94RA29YCH8R0F6BMF",
+		};
+
+		var accountClient = new AccountClient(
+			MockHttpHandler(data),
+			new(DateTimeZoneProviders.Tzdb));
+
+		var details = await accountClient.GetDetails(
```

Comment:
Since the id does not matter here:
```suggestion
		var details = await accountClient.GetDetails(Guid.NewGuid());
```

Reply:
Updated